homeassistant/components/discovergy/sensor.py

```diff
@@ -221,6 +221,7 @@ def __init__(
             name=f"{meter.measurement_type.capitalize()} {meter.location.street} {meter.location.street_number}",
             model=f"{meter.type} {meter.full_serial_number}",
```

Comment:
I think we need to remove the serial number from here in that case?

Reply:
Yes 👍